pset-iterators.js

```diff
@@ -171,6 +243,33 @@ console.log('Problem 6:')
 
 // Add your code below this line
 
+const shortTermPresidents = function(){
+
+  // remove the president who hasn't left office yet
+  presidents.pop()
```

Comment:
@carly-ga interesting approach, remember `.pop()` mutates the original presidents array, in a real application that may not be ideal but in this case, it's fine